frb_example/pure_dart/rust/src/api.rs

```diff
@@ -952,6 +968,19 @@ pub fn naivedatetime(d: chrono::NaiveDateTime) -> chrono::NaiveDateTime {
     d
 }
 
+pub fn optional_empty_datetime_utc(
+    d: Option<chrono::DateTime<chrono::Utc>>,
+) -> Option<chrono::DateTime<chrono::Utc>> {
+    assert_eq!(&d, &None);
+    d
+}
+
+pub fn boxed_empty_datetime_utc(
```

Comment:
```suggestion
pub fn boxed_datetime_utc(
```

ubernit: remove "empty" since looks it is not nullable?